### PR TITLE
Add Dockerfile to templates

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Relates to https://github.com/stackrlabs/cli/pull/55

Closes https://github.com/stackrlabs/sdk/issues/245

### CODE REVIEW

**Comment by kautukkundan** [comment]
do update this in the docs and link it

File: bridge/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+RUN npm install -g npm@latest
+
+# Install bun globally
+RUN npm install -g bun
+
+WORKDIR /app
+
+# Set the NPM token for private package access
+RUN echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >.npmrc
```
[the remark this answers is not in the document]
Comment (aashutoshrathi) [reply bridge/Dockerfile:14]: Yes it's such a major bummer

**Comment by aashutoshrathi** [comment]
Will add these same instructions as here: https://github.com/stackrlabs/cli/pull/55

File: bridge/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (aashutoshrathi) [reply bridge/Dockerfile:14]: Private packages are hard to crack

**Pull request merged**